### DISCUSSION TITLE
Would it be possible for release emails to include a description please?

### DISCUSSION THREAD

**Discussion by claremacrae** [javalent/obsidian-leaflet#258] [Ideas]
Hallo,

As a keen user of this plugin, I subscribe to release notifications.

Here is an example one:

![image](https://user-images.githubusercontent.com/4840096/145617125-0feed181-11b8-40d6-a812-8950ee46c81e.png)

Visiting the "release page" gives this:

![image](https://user-images.githubusercontent.com/4840096/145617183-bb6ba7b9-b360-4e11-a10a-640719d54637.png)

It would be lovely if one of these could one day include a meaningful description to what has changed - or a link to the description.

**Comment by claremacrae** [comment]
Hallo, thanks for experimenting with this, and name-checking me... :-)

The emails are now huge, containing all the changes in every past release - it seems they are emailing the whole change-log, rather than just the changes of the individual release...

I do appreciate that this does contain the info on the current release, which helps... it does mean it's a long way to scroll down to get to the release on GutHub... :-)

And you probably know this, but the list of Releases is unchanged... no description...
<https://github.com/valentine195/obsidian-leaflet-plugin/releases>

I don't know how feasible this is, but I'm thinking based on other projects, the ideal would be to include just the part of the CHANGELOG that describes the individual release - in:

- the email
- the changelog that appears on GitHub (for example <https://github.com/valentine195/obsidian-leaflet-plugin/releases/tag/4.7.1>

If that's not easy, then perhaps the most usable is for the email and the release page to link to CHANGELOG.md ?

**Reply by valentine195** [reply]
yeah, i'm still working on it :) testing it out on a test repo... thought i had it but clearly it didn't work.

**Reply by valentine195** [reply]
The problem is

> the ideal would be to include just the part of the CHANGELOG that describes the individual release

is extremely difficult in `standard-version`, apparently.

**Reply by valentine195** [reply]
That should be better!

**Reply by claremacrae** [reply]
> That should be better!

Yep - 4.7.2 was great in both the email and the release link - very many thanks!
